collector/command.go

```diff
@@ -11,12 +11,41 @@ import (
 )
 
 var (
+	vtyshEnable     = kingpin.Flag("frr.vtysh", "Use vtysh to query FRR instead of each daemon's UNIX socket (default: disabled, recommended: disabled).").Default("false").Bool()
 	vtyshPath       = kingpin.Flag("frr.vtysh.path", "Path of vtysh.").Default("/usr/bin/vtysh").String()
 	vtyshTimeout    = kingpin.Flag("frr.vtysh.timeout", "The timeout when running vtysh commands (default: 20s).").Default("20s").Duration()
 	vtyshSudo       = kingpin.Flag("frr.vtysh.sudo", "Enable sudo when executing vtysh commands.").Bool()
 	frrVTYSHOptions = kingpin.Flag("frr.vtysh.options", "Additional options passed to vtysh.").Default("").String()
 )
 
+func executeBGPCommand(cmd string) ([]byte, error) {
+	if *vtyshEnable {
+		return execVtyshCommand("-c", cmd)
+	}
+	return socketConn.ExecBGPCmd(cmd)
+}
+func executeOSPFCommand(cmd string) ([]byte, error) {
```

Comment:
Missing newline before function.

Reply:
gofmt should be fixing stuff like that.

Reply:
No, neither gofmt nor goimports is that strict on that regard.